src/ssl.c

```diff
@@ -1040,6 +1040,30 @@ int wolfSSL_set_AcceptFilter(
     return WOLFSSL_SUCCESS;
 }
 
+int wolfSSL_CTX_set_ConnectFilter(
+    WOLFSSL_CTX *ctx,
+    NetworkFilterCallback_t ConnectFilter,
+    void *ConnectFilter_arg)
+{
+    if (ctx == NULL)
+        return BAD_FUNC_ARG;
+    ctx->ConnectFilter = ConnectFilter;
+    ctx->ConnectFilter_arg = ConnectFilter_arg;
+    return WOLFSSL_SUCCESS;
```

Comment:
I prefer return 0 = success for our API's. Only compatibility API's should use the `WOLFSSL_SUCCESS`.

Reply:
indeed.  done.

I find this in ssl.h appalling:
```
    WOLFSSL_ERROR_NONE      =  0,   /* for most functions */
    WOLFSSL_FAILURE         =  0,   /* for some functions */
    WOLFSSL_SUCCESS         =  1,
```
srsly? ...

Reply:
Agreed, the source is the openssl compatibility layer... not us. That is why I prefer zero return for success and negatives for errors. Only the compatibility API's should use those above.